sdk/devcenter/azure-developer-devcenter/swagger/README.md

```diff
@@ -0,0 +1,25 @@
+## Generate autorest code
+
+```yaml
+use: '@autorest/java@4.1.5'
+input-file:
+  - https://github.com/Azure/azure-rest-api-specs/blob/75a8d8dcc9f6d0ec626bdeb32f5154f20c8c61cd/specification/devcenter/data-plane/Microsoft.DevCenter/preview/2022-03-01-preview/devcenter.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/75a8d8dcc9f6d0ec626bdeb32f5154f20c8c61cd/specification/devcenter/data-plane/Microsoft.DevCenter/preview/2022-03-01-preview/devbox.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/75a8d8dcc9f6d0ec626bdeb32f5154f20c8c61cd/specification/devcenter/data-plane/Microsoft.DevCenter/preview/2022-03-01-preview/environments.json
```

Comment:
Make sure these swagger is at least approved in the PR (you can ping Laurent or me the PR). We should be able to help if PR is required by SDK arch board.

Reply:
Yeah, that's the correct PR -- apologies, hadn't prioritized it since it's flagged as a breaking change although I believe it is not since the SDK is not generated yet. I've provided the justification in there -- the APIs are merged to main, this PR is just to update the operation IDs based on feedback from the SDK arch board and to remove two invalid query parameters that were mistakenly added (`?top` parameters on two GET calls).

Reply:
PR is approved, thanks Weidong for your help!